<foutmelding>
   <context>Fout tijdens opslaan BRP.</context>
   <stacktrace>java.lang.NullPointerException
       at
       nl.bzk.migratiebrp.synchronisatie.dal.service.impl.relateren.PersoonRelateerderBasis.updateRelatiesVanPersoon(PersoonRelateerderBasis.java:70)
       at
       nl.bzk.migratiebrp.synchronisatie.dal.service.impl.BrpDalServiceImpl.converteerEnRelateerPersoonslijst(BrpDalServiceImpl.java:215)
       at
       nl.bzk.migratiebrp.synchronisatie.dal.service.impl.BrpDalServiceImpl.persisteerPersoonslijst(BrpDalServiceImpl.java:170)
       at sun.reflect.GeneratedMethodAccessor148.invoke(Unknown Source)
       at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
       at java.lang.reflect.Method.invoke(Unknown Source)
       at org.springframework.aop.support.AopUtils.invokeJoinpointUsingReflection(AopUtils.java:319)
       at
       org.springframework.aop.framework.ReflectiveMethodInvocation.invokeJoinpoint(ReflectiveMethodInvocation.java:183)
       at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:150)
       at org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:110)
       at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:172)
       at org.springframework.aop.framework.JdkDynamicAopProxy.invoke(JdkDynamicAopProxy.java:202)
       at com.sun.proxy.$Proxy51.persisteerPersoonslijst(Unknown Source)
       at nl.bzk.migratiebrp.test.common.ConversieTestCasus.testOpslaanBrp(ConversieTestCasus.java:233)
       at nl.bzk.migratiebrp.test.db.DbConversieTestCasus.run(DbConversieTestCasus.java:132)
       at nl.bzk.migratiebrp.test.common.runner.TestRunner.runTest(TestRunner.java:128)
       at nl.bzk.migratiebrp.test.common.runner.TestRunner.run(TestRunner.java:114)
       at nl.bzk.migratiebrp.test.common.runner.TestRunner.run(TestRunner.java:118)
       at nl.bzk.migratiebrp.test.common.runner.TestRunner.run(TestRunner.java:106)
       at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:50)
       at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
       at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:467)
       at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:683)
       at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:390)
       at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:197)
   </stacktrace>
</foutmelding>
